packages/settings/src/lib/boson.ts

```diff
@@ -4,6 +4,8 @@ const CUREXA_ID = 'phr_01GCA54GVKA06C905DETQ9SY98';
 const CAREPOINT_ID = 'phr_01GA9HPVBVJ0E65P819FD881N0';
 // const ALTO_ID = 'phr_01G9CM93X1NFP1C9H9K50DPKHX';
 const AMAZON_PHARMACY_ID = 'phr_01GA9HPV5XYTC1NNX213VRRBZ3';
+const TRUE_PILL = 'phr_01HHDHKAMNMKC3CRY8VCYRVVPE';
```

Comment:
nit: mind naming TRUE_PILL_ID just for consistency?

Reply:
done